client/src/components/NavBar/NavBar.spec.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react'
+import { BrowserRouter as Router } from 'react-router-dom'
+import NavBar from './NavBar'
+
+describe('Navigation', () => {
+  it('renders the navigation', () => {
+    render(
+      <Router>
+        <NavBar />
+      </Router>
+    )
+
+    const navigation = screen.getByRole('navigation')
+    expect(navigation).toBeInTheDocument()
+  })
+
+  it('has at leat one nav link', () => {
+    render(
+      <Router>
+        <NavBar />
+      </Router>
+    )
+    const links = screen.getAllByRole('link')
+    expect(links.length).toBeGreaterThan(0)
+  })
```

Comment:
Why don't you just test the exact number of links in your navigation? You know how many there should be, don't you?